src/CoreShop/Component/Index/Getter/FieldCollectionGetter.php

```diff
@@ -44,9 +52,31 @@ public function get(IndexableInterface $object, IndexColumnInterface $config): m
         }
 
         foreach ($validItems as $item) {
-            if (method_exists($item, $fieldGetter)) {
-                $fieldValues[] = $item->$fieldGetter();
+            /**
+             * @var Localizedfields|null $localizedFieldsFd
+             */
+            $localizedFieldsFd = $item->getDefinition()->getFieldDefinition('localizedfields');
+            $fd = $item->getDefinition()->getFieldDefinition($config->getObjectKey());
+            $localizedFd = $localizedFieldsFd?->getFieldDefinition($config->getObjectKey());
+
+            if (!$fd) {
+                continue;
             }
+
+            if (!method_exists($item, $fieldGetter)) {
+                continue;
+            }
+
+            if ($localizedFd) {
+                LocaleFallbackHelper::useFallbackValues(function() use ($item, $fieldGetter, &$fieldValues) {
+                    foreach ($this->localeProvider->getDefinedLocalesCodes() as $locale) {
+                        $fieldValues[$locale][] = $item->$fieldGetter($locale);
```

Comment:
```suggestion
                        $fieldValues[$locale] = $item->$fieldGetter($locale);
```
It doesn't work with [] for me. Can you test it one more time?